daemon/container_operations_windows.go

```diff
@@ -35,6 +40,58 @@ func detachMounted(path string) error {
 	return nil
 }
 
+func (daemon *Daemon) setupSecretDir(c *container.Container) (setupErr error) {
+	if len(c.SecretReferences) == 0 {
+		return nil
+	}
+
+	localMountPath := c.SecretMountPath()
+	logrus.Debugf("secrets: setting up secret dir: %s", localMountPath)
+
+	defer func() {
+		if setupErr != nil {
+			if err := os.RemoveAll(localMountPath); err != nil {
+				if !os.IsNotExist(err) {
+					logrus.Errorf("error cleaning up secret mount: %s", err)
+				}
+			}
+		}
+	}()
+
+	// create local secret root
+	if err := system.MkdirAllWithACL(localMountPath, 0); err != nil {
+		return errors.Wrap(err, "error creating secret local directory")
+	}
+
+	for _, s := range c.SecretReferences {
+		if c.SecretStore == nil {
+			return fmt.Errorf("secret store is not initialized")
+		}
+
+		// TODO (ehazlett): use type switch when more are supported
```

Comment:
these should be fine to remove as the implementation will know to do it.  there were some timeline and roadmap adjustments that will delay this a bit so I think it's fine to remove for now.